docs/yaml-preprocessing.md

```diff
@@ -418,6 +418,18 @@ things: !$split
 #   - b
 ```
 
+* `!$join` join a list into a string
```

Comment:
Let's also mention why we are duplicating what's in the CFN intrinsic functions. When would you use this (i.e. in `iidy render`) vs `!Join`